web/profiles/custom/os2loop/modules/os2loop_upvote/modules/os2loop_upvote_fixtures/os2loop_upvote_fixtures.info.yml

```diff
@@ -0,0 +1,12 @@
+name: 'OS2Loop Upvote fixtures'
+type: module
+description: 'OS2Loop Upvote fixtures'
+# Used only for development and testing.
+hidden: true
+core_version_requirement: ^9
+package: OS2Loop
+# version: VERSION
+dependencies:
+  - os2loop_fixtures
+  - os2loop_question
+  - os2loop_post
```

Comment:
The fixtures module should have a dependency on the `os2loop_update` module (and not on `os2loop_question` and `os2loop_post`)